sammba/registration/struct.py

```diff
@@ -9,9 +9,10 @@
 def anats_to_common(anat_filenames, write_dir, brain_volume,
                     registration_kind='affine',
                     nonlinear_levels=[1, 2, 3],
-                    nonlinear_minimal_patch=75,
-                    convergence=0.005, caching=False, verbose=False,
-                    unifize_kwargs=None, brain_extraction_unifize_kwargs=None):
+                    nonlinear_minimal_patches=[75],
+                    nonlinear_weight=None,
```

Comment:
From what I understand (please correct me if wrong), the weight is a binary mask (0 or 1), which needs to include brain and some scalp. May be naming it as `nonlinear_mask_filename` is better, because weight is not very informative to non AFNI users.

Reply:
It is a weight, it does not have to be binary